src/test/java/com/es/phoneshop/model/product/ArrayListProductDaoTest.java

```diff
@@ -3,19 +3,54 @@
 import org.junit.Before;
 import org.junit.Test;
 
-import static org.junit.Assert.assertTrue;
+import java.math.BigDecimal;
+import java.util.Currency;
+import java.util.List;
+
+import static org.junit.Assert.*;
 
 public class ArrayListProductDaoTest
 {
     private ProductDao productDao;
+    private  Product product;
 
     @Before
     public void setup() {
         productDao = new ArrayListProductDao();
+        Currency usd = Currency.getInstance("USD");
```

Comment:
Better to make is private class field to avoid code duplication in testDeleteProduct().